modules/installation-aws-marketplace-subscribe.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-user-infra.adoc
+
+ifeval::["{context}" == "installing-aws-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-aws-government-region"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-aws-user-infra"]
+:upi:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-aws-marketplace-subscribe_{context}"]
+= Obtaining an AWS Marketplace image
+If you are deploying an {product-title} cluster using an AWS Marketplace image, you must first subscribe through AWS. Subscribing to the offer provides you with the AMI ID that the installation program uses to deploy worker nodes.
+
+[IMPORTANT]
+====
+Deploying an {product-title} cluster using an AWS Marketplace image is not supported in Top Secret regions. 
```

Comment:
```suggestion
Deploying an {product-title} cluster using an AWS Marketplace image is not supported in secret regions. 
```
Deploying an {product-title} cluster using an AWS Marketplace image is not supported in secret regions.

Reply:
Sounds good to me.